common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -166,6 +166,81 @@ def test_demand_hint(self):
             actual_events)
 
 
+class ProblemHintWithHtmlTest(ProblemsTest, EventsTestMixin):
+    """
+    Test that extended hint features plumb through to the page html and tracking log.
+    """
+
+    def get_problem(self):
+        """
+        Problem with extended hint features.
```

Comment:
Explain here why this problem HTML is interesting - ie why are you defining yet more?